plugin/src/test/java/io/jenkins/plugins/analysis/warnings/steps/JobActionITest.java

```diff
@@ -196,6 +197,34 @@ void shouldHaveSidebarLinkEvenWhenLastActionHasNoResults() {
         assertThat(jobAction).isNotNull();
     }
 
+    /**
+     * Verifies the behaviour of {@link AnalysisBuildsFallback}.
+     */
+    @Test
+    void shouldAttachJobActionsWithoutRecordIssues() {
+        FreeStyleProject project = createFreeStyleProjectWithWorkspaceFilesWithSuffix(ECLIPSE_LOG);
+        enableEclipseWarnings(project);
+
+        Run<?, ?> firstBuild = buildWithResult(project, Result.SUCCESS);
+        List<ResultAction> firstBuildResultActions = firstBuild.getActions(ResultAction.class);
+        assertThat(firstBuildResultActions).isNotEmpty();
+
+        List<JobAction> jobActionsAfterFirstBuild = project.getActions(JobAction.class);
+        assertThat(jobActionsAfterFirstBuild).isNotEmpty();
+
+        project.getPublishersList().clear();
```

Comment:
Can you start another build with action so that the trend appears? Then we can check for the trend chart as well...